Source/SwiftLintFramework/Rules/RuleConfigurations/NumberSeparatorConfiguration.swift

```diff
@@ -33,6 +35,13 @@ public struct NumberSeparatorConfiguration: RuleConfiguration, Equatable {
             self.minimumFractionLength = minimumFractionLength
         }
 
+        if let validRanges = configuration["valid_ranges"] as? [[String: Any]] {
```

Comment:
I know that I kind of suggested the name, but maybe we can think of about a better name for this? `no_separator_ranges` maybe?

Reply:
I don't think `no_separator_ranges` explains it well, as it suggests that no separators can be used for the values in the specified ranges. The option was meant to allow users both, separators or no separators depending on the semantic meaning of the number. For examples `2_048` could be used for 2^11 but `2048` could be used for the year. Both should be possible and what the range basically does is turning off the rule for the given ranges. So I'm proposing `exclude_ranges` instead. I've changed the code accordingly.

Reply:
ok, I had misunderstood the change before. This makes sense.